examples/router/main.go

```diff
@@ -65,7 +65,7 @@ func main() {
 			os.Exit(1)
 		}
 		return
-	case <-time.After(time.Second):
+	case <-time.After(10 * time.Second):
```

Comment:
Why 10 seconds? It only needs to send 3 messages so I think 1 second should be plenty. I think we would want the CI to fail if it took more than a second.

Reply:
@ross-pure yeah that was me tweaking something to see why the test wasn't passing originally. Forgot to change it back.